server/src/controllers/game.controller.js

```diff
@@ -100,6 +106,33 @@ class GameController {
       throw error;
     }
   }
+
+  async saveMoveToDb(req, res){
```

Comment:
Save move to database can't be a middleware.
The argument you are passing to it above is not `res` and `req`